doc/source/reference/api/grn_expr.rst

```diff
@@ -48,51 +48,6 @@ Reference
 
 .. c:function:: grn_rc grn_expr_append_op(grn_ctx *ctx, grn_obj *expr, grn_operator op, int nargs)
 
-.. c:function:: grn_rc grn_expr_get_keywords(grn_ctx *ctx, grn_obj *expr, grn_obj *keywords)
-
-   Extracts keywords from ``expr`` and stores to
-   ``keywords``. Keywords in ``keywords`` are owned by ``expr``. Don't
-   unlink them. Each keyword is ``GRN_BULK`` and its domain is
-   ``GRN_DB_TEXT``.
```

Comment:
Can we keep this information?

Reply:
fix: [fa3e05f](https://github.com/groonga/groonga/pull/2264/commits/fa3e05f9e4473d4928151207cf983c490feba8cc)